spec/fixtures/project-content-with-prettier-errors-for-custom-options/index.js

```diff
@@ -0,0 +1,3 @@
+import './es6-extension'
```

Comment:
We also don't need to have this extensive testing. The different file names are not necessary, given they are tested in the original prettier integration test. A single `index.js` file should be enough. Wdyt?